test_graphs.py

```diff
@@ -2,18 +2,22 @@
 import pytest
 
 from run_config_input import run_normal_config_file
-from utils.random_graph_data import random_graph_data
+from utils.deterministic_graph_data import deterministic_graph_data
 from utils.utils import get_comm_size_and_rank
 from test_trained_model import test_trained_model
 
+import torch
+
+torch.manual_seed(0)
+
 
 @pytest.mark.parametrize("model_type", ["GIN", "GAT", "MFC", "PNN"])
```

Comment:
MFC fails on the assert if run first - I don't understand why, but that's concerning

Reply:
MFC fails in my local runs. Any difference between running locally and here?

Reply:
Hopefully just random fluctuations then - I get `0.1134`. Maybe we should just use different tolerances for each